app/views/wizard/index.html.erb

```diff
@@ -18,7 +18,7 @@
 
     <div id="svg-sprites" style="display:none;">
       <div class="fontawesome">
-        <%= raw SvgSprite.bundle %>
+        <%= raw SvgSprite.bundle([]) %>
```

Comment:
I agree with @pmusaraj that this seems odd to me. I understand the rationale that you almost always want to call it with a value, but we have at least this one example of not doing that.

If you want to keep the required parameter, then I would suggest adding a new method `empty_bundle` or something for this particular case (and maybe in your tests too.)

Reply:
:+1: ok, I'm persuaded - I've added a default value, and removed all the helper stuff I added to the tests